Python/README.md

```diff
@@ -13,6 +13,16 @@
 
 _add list here_
 
+## Other
+* [Check Inversion](other/Check_Inversion.py)
+* [Kadane's Algorithm](other/Kadanes_Algorithm.py)
+* [Merge two arrays](other/merge_sorted_arrays.py)
+* [Reverse a string using stack](other/Reverse_string_using_stack.py)
```

Comment:
Manually edit the readme file to link your program, don't make changes to any existing links.

Reply:
I added it manually, actually there were so many repeated files, i tally them with TesseractCoding master branch and remove those files which were not present in the directory but added in the readme file.

Reply:
Revert the changes and just add your program link in the end of the Other heading by editing the README.md file.

Reply:
> Revert the changes and just add your program link in the end of the Other heading by editing the README.md file.

But sir, if i do so, then readme file will have repetitive blocks of read me file in itself. Before making changes into readme file, i saw there were so repetitive blocks of files.

Reply:
I have formatted README.md. Update your branch and add the link of your program under appropriate heading.

Reply:
> I have formatted README.md. Update your branch and add the link of your program under appropriate heading.

Done Sir